### PR TITLE
Crm 8 avatar component

### PR DESCRIPTION
**Pull request by magikabdul**

### CODE REVIEW

File: frontend/src/components/atoms/Avatar/Avatar.stories.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      width: '32px',
+      height: '32px',
+    }}
+    >
+      <Avatar {...args} />
+    </div>
+  </>
+);
+
+export const LargeAvatar = TemplateLarge.bind({});
```
Comment (Fellway) [inline frontend/src/components/atoms/Avatar/Avatar.stories.js:47]: You can use arrow function:
export const LargeAvatar = () => <TemplateLarge imageLink={imageLink} />;

File: frontend/src/components/atoms/Avatar/Avatar.stories.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import Avatar from './Avatar';
+
+import imageLink from '../../../assets/images/flower.jpg';
+
+export default {
+  title: 'Atoms/Avatar',
+  component: Avatar,
+};
+
+const TemplateLarge = (args) => (
```
Comment (Fellway) [inline frontend/src/components/atoms/Avatar/Avatar.stories.js:11]: Please use decorator:
https://storybook.js.org/docs/react/writing-stories/introduction

File: frontend/src/components/atoms/Avatar/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  background-image: ${({ imageLink }) => `url(${imageLink})`};
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  
+  overflow: hidden;
+`;
+
+function Avatar({ imageLink }) {
```
Comment (Fellway) [inline frontend/src/components/atoms/Avatar/Avatar.js:18]: Use arrow function

File: frontend/src/components/atoms/Avatar/Avatar.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <Container imageLink={imageLink} />
+  );
+}
+
+Avatar.propTypes = {
+  imageLink: PropTypes.string.isRequired,
+};
+
+Avatar.defaultProps = {};
```
Comment (Fellway) [inline frontend/src/components/atoms/Avatar/Avatar.js:28]: It's not needed

**Review by Fellway — CHANGES_REQUESTED** [review]

File: frontend/src/components/atoms/Avatar/Avatar.js (hunk 3 above)
Comment (magikabdul) [reply frontend/src/components/atoms/Avatar/Avatar.js:18]: changed to arrow function

File: frontend/src/components/atoms/Avatar/Avatar.js (hunk 4 above)
Comment (magikabdul) [reply frontend/src/components/atoms/Avatar/Avatar.js:28]: removed

File: frontend/src/components/atoms/Avatar/Avatar.stories.js (hunk 1 above)
Comment (magikabdul) [reply frontend/src/components/atoms/Avatar/Avatar.stories.js:47]: used

File: frontend/src/components/atoms/Avatar/Avatar.stories.js (hunk 2 above)
Comment (magikabdul) [reply frontend/src/components/atoms/Avatar/Avatar.stories.js:11]: decorator used

**Pull request closed without merging**